R/sql_methods.R

```diff
@@ -204,6 +204,9 @@ get_credentials <- function(username, password, dbname, schemaname){
   if (is_local()){
     return(get_local_info(username, password, dbname, schemaname))
   } else {
-    return(get_nuvolos_info(username, password, dbname, schemaname))
+    cred <- get_nuvolos_info(username, password, dbname, schemaname)
+    # getting rid of the extra quotation marks around the dbname and schemaname
+    return(lapply(cred, function(x) if (length(unlist(strsplit(x,'"')))>1) {unlist(strsplit(x,'"'))[2]} else {x})
```

Comment:
I think we should only call this for the dbname / schemaname, if I see currently it's also applied for password, which might(?) contain a "

Reply:
The password is the nuvolos-genarated password for the instances, I don't think they can contain ". But I'll correct it anyway, just to be sure.